### DISCUSSION TITLE
How to open vips after brew?

### DISCUSSION THREAD

**Discussion by meliiissa** [libvips/libvips#2065] [Q&A]
Ask a question...
Hey,

I'm a super newbie, so I just installed brew and then vips (MacOs). Everything worked fine, but I don't understand how to open vips now? I need to install vips for my computer engineering class. My prof uses windows and he said that there's no other options than windows. Our computer rooms are closed due covid and I just can't afford a new windows computer. I don't want to pay 100€ for windows license :( 

Sorry for my bad english, I look forward to receiving your continued support. 

melissa.

**Comment by jcupitt** [comment]
Hi @meliiissa,

You've installed it, so just type `vips` in the terminal to run it. Do you need to use Python with it?
